authors/apps/authentication/views.py

```diff
@@ -115,12 +115,27 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
 
-        return Response(serializer.data, status=status.HTTP_200_OK)
+        recipient = serializer.data['email']
+        subject = 'Activate Authors Haven Account'
+        host = 'http://' + request.get_host()
+        url = '/api/password-reset/'
+        token = serializer.data['token']
+        content = "Follow this link to Reset your"
```

Comment:
Replace these two lines with

```
content = "Follow this link to Reset your account password {}{}{}/".format(
    host, url, token
)
```
It's much easier to read.

Reply:
alright